psi4/driver/procrouting/sapt/sapt_util.py

```diff
@@ -144,10 +144,12 @@ def print_sapt_dft_summary(data, name, short=False):
     core.set_variable("SAPT IND ENERGY", ind)
 
     # Dispersion
-    disp = data["Disp20"] + data["Exch-Disp20,u"]
+    # disp = data["Disp20"] + data["Exch-Disp20,u"]
+    disp = data["Disp20"] + data["Exch-Disp20,r"]
     ret += print_sapt_var("Dispersion", disp) + "\n"
     ret += print_sapt_var("  Disp2,r", data["Disp20"]) + "\n"
     ret += print_sapt_var("  Disp2,u", data["Disp20,u"]) + "\n"
+    ret += print_sapt_var("  Exch-Disp2,r", data["Exch-Disp20,r"]) + "\n"
     ret += print_sapt_var("  Exch-Disp2,u", data["Exch-Disp20,u"]) + "\n"
```

Comment:
is `Exch-Disp20,u` still available to print?

Reply:
Yes, it's right in the next line.

Reply:
I meant is the _data_ available for the next line to print or is it going to print the initialized zero?

Reply:
Yes it is available. See lines 324 to 327 in `sapt_mp2_terms.py`.